src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,131 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import core.basesyntax.dao.StorageDao;
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static StorageDao storage;
+    private static RegistrationService registrationService;
+
+    @BeforeAll
+    static void beforeAll() {
+        storage = new StorageDaoImpl();
+        registrationService = new RegistrationServiceImpl();
+        storage.add(registrationService.register(new User("HeroWizard", "Qwerty123!", 26)));
+        storage.add(registrationService.register(new User("mrDiller227", "LoveYourMum!337", 29)));
```

Comment:
do not use your methods to create test data. Just hardcode it 
```suggestion
        storage.add(new User("HeroWizard", "Qwerty123!", 26));
        storage.add(new User("mrDiller227", "LoveYourMum!337", 29));
```